src/routers/controllers/resendEmailController.ts

```diff
@@ -1,6 +1,6 @@
 import { Request, Response } from "express";
 import * as constants from "../../constants";
-import { isEmailAuthorised } from "../../services/userCompanyAssociationService";
+import { isEmailAuthorised } from "../../services/associationsService";
 import { getExtraData, setExtraData } from "../../lib/utils/sessionUtils";
 import { sendAuthorisationEmail } from "../../services/emailNotificationService";
```

Comment:
what does this do?
i though emails will be sent by backend.

Reply:
At the moment it does nothing, just returns hard coded status 201. Although the job of resending email notification may be done by the backend once it's implemented, the frontend has to initiate the whole process when the appropriate link is clicked and then display confirmation that the email has been resent based on the response from the backend. So at the moment this is just a skeleton function to be filled with logic once this part of the system is ready.

Reply:
this should call the invitation endpoint and back end will take care of things.

Reply:
At the time this code was created, there was nothing to call and we had a very vague understanding of how this feature is supposed to work. I assume there is an integration ticket now that will allow us to add the missing logic.